docs/Model_Assumptions.rst

```diff
@@ -63,6 +63,61 @@ In case of excessive excess energy, a warning is given that it seems to be cheap
 High excess energy can for example result into an optimized inverter capacity that is smaller than the peak generation of installed PV.
 This becomes unrealistic when the excess is very high.
 
+Stratified thermal storage
+##########################
+
+In order to model a stratified thermal energy storage the two parameters `fixed_losses_relative` and `fixed_losses_absolute` are passed through `storage_xx.csv`.
+They are used to take into account temperature dependent losses of a thermal storage. To model a thermal energy storage without stratification, the two parameters are not set. The default values of `fixed_losses_relative` and `fixed_losses_absolute` are zero.
+Except for these two additional parameters the stratified thermal storage is implemented in the same way as other storage components.
+
+Precalculations of the `installedCap`, `efficiency`, `fixed_losses_relative` and `fixed_losses_absolute` can be done orientating on the stratified thermal storage component of `oemof.thermal  <https://github.com/oemof/oemof-thermal>`__.
+Further parameters, which can be precalculated, are: U-value, volume and surface of the storage.
```

Comment:
Or which are precalculated? Or which are needed to define above parameters in another pre-processing step?

Reply:
Done with commit https://github.com/rl-institut/multi-vector-simulator/pull/718/commits/4ec3483a5c859f27601e7a5ba5dbc8280157f2dd